app/src/main/res/values/strings.xml

```diff
@@ -44,6 +44,8 @@
     <string name="error_username_invalid">Username has invalid characters</string>
     <string name="error_password_invalid">Password has invalid characters</string>
     <string name="error_vnc_password_invalid">VNC Password has invalid characters</string>
+    <string name="error_username_invalid_characters">Characters in username must be 1 to 30 characters with lowercase letters and/or numbers.</string>
+    <string name="error_username_in_blacklist">The username: `%1$s` is reserved and cannot be used as a filesystem username.</string>
```

Comment:
Are the backticks surrounding the substitution just for clarity, or do they do something in XML?

Reply:
The back ticks were just for clarity but I've removed it in favor of a static string, which makes it easier to test